### DISCUSSION TITLE
How to convert tess file into an obj file

### DISCUSSION THREAD

**Discussion by Geo572** [neperfepx/neper#658] [Q&A] [answered]
Hello Everyone,
I'm new here to use Neper, please help me regarding this issue.

I use the below commands but I don't understand how to convert tess file into an obj file

neper -T -n 100 -morpho gg -reg 1

neper -T -n 100 -morpho "diameq:lognormal(1,0.05),1-sphericity:lognormal(0.145,0.03)" -domain "square(0.06,0.12)" -dim 2

Thank you.

**Comment by rquey** [comment, marked as answer]
To convert an existing tess file:
```
neper -T -loadtess <file_name> -format obj
```

Otherwise, just add `-format obj` (or `-format tess,obj`) to your `-T` command.

**Reply by Geo572** [reply]
Thank you @rquey,
I give the command neper -T -loadtess <n100-id1.tess> -format obj  it show me error "Aborted (core dumped).
Can you give the example of file name?
